packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -262,6 +262,22 @@ class ScaffoldMessengerState extends State<ScaffoldMessenger> with TickerProvide
   ///
   /// ** See code in examples/api/lib/material/scaffold/scaffold_messenger_state.show_snack_bar.0.dart **
   /// {@end-tool}
+  ///
+  /// ## Troubleshooting
+  ///
+  /// ### Why isn't my floating [SnackBar] visible ?
```

Comment:
This issue does not appear common, instead, can we make this more informative to the relative positioning of the SnackBar and the floating action button?


Maybe something along the lines of...

```suggestion
  /// ## Relative positioning of floating SnackBars
```

Reply:
Done